client/my-sites/sharing/connections/mailchimp-settings.jsx

```diff
@@ -88,6 +88,19 @@ const MailchimpSettings = ( {
 						</option>
 					) ) }
 			</select>
+			{ mailchimpLists && mailchimpLists.length > 0 && mailchimpListId !== 0 && (
+				// eslint-disable-next-line wpcalypso/jsx-classname-namespace
+				<div className="sharing-connections__mailchimp-gutenberg_explanation">
+					<p>
+						{ translate(
+							'Start building your mailing list by adding the Mailchimp block to your posts and pages. '
+						) }
+						<a href={ translate( 'https://support.wordpress.com/mailchimp-block/' ) }>
```

Comment:
I don't think we need to wrap this URL in `translate` tho? The site itself handles redirections to different languages.

Looking at other support links around Calypso, looks like they use `_blank` targets, so:
```jsx
<ExternalLink
			target="_blank"
			href="https://support.wordpress.com/mailchimp-block/"
>
			{ translate( 'Learn more.' ) }
</ExternalLink>
```